Status.sh

```diff
@@ -80,7 +80,7 @@ repo="\Zb\Z1$(< /srv/repo)\Zn" # Set the color to red if the repository name is
 fi
 
 branch="$(< /srv/brnch)"
-if [ ! "$(< /srv/brnch)" = "vps-1" ] || [ ! "$(< /srv/brnch)" = "vps-dev" ]
+if [ ! "$(< /srv/brnch)" = "vps-1" ] && [ ! "$(< /srv/brnch)" = "vps-dev" ]
```

Comment:
Why was this changed?

Reply:
It was a mistake.
If the branch name is vps-1 or if the branch name is vps-dev, I want the color to be normal.

But, I have inverted it.  So, I should have used the Demorgan's law and changed or to and after inverting.  But, I didn't.  So, it is broken now.  It needs to be And.

Reply:
If you update to the existing vps-dev branch, vps-dev on your status page will be shown in red.